resources/definitions/fdmprinter.def.json

```diff
@@ -4919,6 +4919,17 @@
                     "enabled": "magic_spiralize",
                     "settable_per_mesh": false,
                     "settable_per_extruder": false
+                },
+                "relative_extrusion":
+                {
+                    "label": "Use Relative Extrusion",
+                    "description": "Use relative extrusion rather than absolute extrusion. Before, Cura always used absolute extrusion. Most printers can cope with either relative or absolute extrusion.",
```

Comment:
Rounding errors in each relative extrusion amount add up. So at the end of the print, you are less sure of the actual amount of material extruded.